tools/login/actions/LoginAction.php

```diff
@@ -204,6 +204,13 @@ private function login()
                 throw new LoginException(_t('LOGIN_WRONG_PASSWORD'));
             }
             $remember = filter_input(INPUT_POST, 'remember', FILTER_VALIDATE_BOOL);
+            if (($this->wiki->GetConfigValue ("signup_mail_activation") === "1") && !$this->userManager->isActivated ($user["name"])){
+                throw new LoginException("Your account must be activated first. " . (
```

Comment:
```suggestion
                throw new LoginException(_t('LOGINACTION_ACTIVATION' . (
```

et mettre dans le fichier de langue `tools/login/lang/login_fr.inc.php`
```php
  'LOGINACTION_ACTIVATION' => "Your account must be activated first. ",
```